api/v1/webspherelibertyapplication_types.go

```diff
@@ -434,10 +434,20 @@ type WebSphereLibertyApplicationRoute struct {
 
 type WebSphereLibertyApplicationSemeruCloudCompiler struct {
 	// Resource requests and limits for the Semeru Cloud Compiler.
-	// +operator-sdk:csv:customresourcedefinitions:order=48,type=spec,displayName="Resource Requirements",xDescriptors="urn:alm:descriptor:com.tectonic.ui:resourceRequirements"
+	// +operator-sdk:csv:customresourcedefinitions:order=48,type=spec,displayName="Enabled",xDescriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
This can be removed as discussed. The comment above is for the `resources` field below (must have been moved up while merging)